R/grid_code_paths.R

```diff
@@ -35,6 +35,7 @@ tune_grid_loop <- function(resamples, grid, workflow, metrics, control, rng) {
           ns = "tune"
         )
 
+        # Likely want to debug at `tune_grid_loop_iter`
```

Comment:
```suggestion
        # Likely want to debug with `debugonce(tune_grid_loop_iter)`
```

How about this, which contains the exact thing you can copy and paste?

Reply:
You got it. :)